lib/rust/parser/tests/parse.rs

```diff
@@ -75,8 +75,7 @@ fn parentheses_nested() {
 
 #[test]
 fn comments() {
-    // Basic, full-line comment.
-    test("# a b c", block![(Comment "# a b c")]);
+    test("# a b c", block![() ()]);
```

Comment:
Comments will not be represented in the `Tree` at all.

Reply:
They're there. They're part of the newlines.

We have different kinds of comments: Plain comments and doc comments. Doc comments are represented in the tree because we need to know their contents. All we need to do with plain comments is ignore them when parsing, and preserve them (in the correct positions) when editing. This is the same thing we need to do with whitespace.